lib/sip-parser/parser.js

```diff
@@ -243,6 +243,25 @@ function parseUri(s) {
         .map(function(s) { return s.split('='); })
         .reduce(function(params, x) { params[x[0]] = x[1]; return params; }, {})
     } ;
+  } else {
+    // try if this is tel format
+    return parseTelUri(s);
+  }
+}
+
+function parseTelUri(s) {
+  if (typeof s === 'object')
+    return s;
+
+  const re = /^(tel):([+]?[0-9-.()]*)(;phone-context=(.*))?$/;
+  const r = re.exec(s);
+
+  if (r) {
+    return {
```

Comment:
In addition to 'context' you should parse and return 'params'.  Please review the ABNF syntax [here](https://datatracker.ietf.org/doc/html/rfc3966#section-3).

Reply:
Please check again, I hope I understand you correctly

Reply:
No, not quite.  In the spec, "context" is called out as a specific property and should be returned as such (like you were doing before).  But additionally, the spec provides for other, general params, and these (if present) should be returned in the 'params' object

Reply:
fixed